common/lib/xmodule/xmodule/js/src/video/09_video_caption.js

```diff
@@ -1137,6 +1139,8 @@
                     .removeClass('is-active')
                     .find('.control-text')
                         .text(gettext('Turn on closed captioning'));
+
```

Comment:
Gabe Mulby said as much. However, for the sake of syncing things up, I'm going to move the emitter to the end of the function.